cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config.go

```diff
@@ -86,29 +78,29 @@ func (clusterConfig *KurtosisClusterConfig) GetClusterType() KurtosisClusterType
 func getSuppliers(clusterId string, clusterType KurtosisClusterType, kubernetesConfig *v2.KubernetesClusterConfigV2) (
 	kurtosisBackendSupplier,
 	engine_server_launcher.KurtosisBackendConfigSupplier,
-	*engine_server_launcher.KurtosisRemoteBackendConfigSupplier,
 	error,
 ) {
-	kurtosisRemoteBackendConfigSupplier := engine_server_launcher.NewKurtosisRemoteBackendConfigSupplier(store.GetContextsConfigStore().GetCurrentContext)
-
 	var backendSupplier kurtosisBackendSupplier
 	var engineConfigSupplier engine_server_launcher.KurtosisBackendConfigSupplier
 	switch clusterType {
 	case KurtosisClusterType_Docker:
 		if kubernetesConfig != nil {
-			return nil, nil, nil, stacktrace.NewError(
+			return nil, nil, stacktrace.NewError(
 				"Cluster '%v' defines cluster config, but config must not be provided when cluster type is '%v'",
 				clusterId,
 				clusterType.String(),
 			)
 		}
 
 		backendSupplier = func(_ context.Context) (backend_interface.KurtosisBackend, error) {
-			kurtosisRemoteBackendConfigMaybe, err := kurtosisRemoteBackendConfigSupplier.GetOptionalRemoteConfig()
-			if err != nil {
-				return nil, stacktrace.Propagate(err, "Error building optional remote Kurtosis backend config")
+			var remoteBackendConfigMaybe *configs.KurtosisRemoteBackendConfig
```

Comment:
This remote backend config is generated from the remote context and used by the CLI to connect to the bastion docker daemon.  A similar object is also used by the engine to connect to the remote backend docker daemon.

Reply:
👍  that makes sense to me, and it's nice we're able to use the same kind of object in those 2 places